### ISSUE TITLE
Event on a bubble/comment resize

### ISSUE THREAD

**Issue by koenvanwijk** [google/blockly#7175]
### Check for duplicates

- [X] I have searched for similar issues before opening a new one.

### Problem

I like to know when a bubble is resized, so I can change the content of a bubble. 

### Request

Please fire an event on a bubble/comment resize. 

### Alternatives considered

On every event save the block (including the comment) and look at the json saved state to see if the comment size is changed.

**Comment by maribethb** [comment]
This needs some investigation into how much work it would be & if it would cause too much spam for people listening to events. It would be a UI event though so can be ignored by a lot of event listeners. If we don't have any reason not to do it, this seems like a reasonable feature request but probably not able to be prioritized by us soon, so we'd accept a PR for this if you'd like to make one.

**Comment by Aarya01Patil** [comment]
Hey I would love to solve this issue @BeksOmega

**Comment by BeksOmega** [comment]
@gonfunko is this fixed actually?

**Comment by gonfunko** [comment]
No, I added support in scratch-blocks but not in core; only workspace comments fire resize events in core.
